### DISCUSSION TITLE
Extract field properties from a pdf file

### DISCUSSION THREAD

**Discussion by mohitchawda** [jsvine/pdfplumber#671] [Q&A]
How to extract txt field properties from a pdf file? e.g. whether field is mandatory or not , any character limit or any javascript validation added as part of filed properties

**Comment by jsvine** [comment]
Hi @mohitchawda, and thanks for your interest in this library. Unfortunately, I don't think I understand this question. Could you provide a specific PDF and some more details about what you're hoping to achieve?
